charts/quine-enterprise/Chart.yaml

```diff
@@ -3,6 +3,6 @@ name: quine-enterprise
 description: A Helm chart for creating a thatDot Quine Enterprise cluster
 type: application
 
-version: 0.4.5
+version: 0.4.6
 
 appVersion: "1.8.1"
```

Comment:
The `appVersion` should be bumped up to `1.9.1`, as it is the first version which supports the `USE_NGINX` and the `USE_BASIC_AUTH` environment variables.

Reply:
I was wondering what the difference between `version` and `appVersion` was 🧠

Reply:
Yep, you can see `appVersion` is used internally by the chart to determine which Docker image tag to pull by default. Of course you can override it, but this way it's documented which version of the chart you are supposed to use for which version of the application.